src/GHC/Util/SrcLoc.hs

```diff
@@ -25,6 +37,12 @@ newtype SrcSpanD = SrcSpanD SrcSpan
   deriving (Outputable, Eq)
 instance Default SrcSpanD where def = SrcSpanD noSrcSpan
 
+newtype FastStringD = FastStringD FastString
```

Comment:
Does GHC not provide an Ord for this type? Could we get one added upstream for the future?

Reply:
@ndmitchell  the `Ord` instance has been removed. see https://gitlab.haskell.org/ghc/ghc/-/commit/4b4fbc58d37d37457144014ef82bdd928de175df. however i overlooked the existence of `LexicalFastString` (see https://hackage.haskell.org/package/ghc-9.2.1/docs/GHC-Data-FastString.html#t:LexicalFastString) which we can use here. pushing a fix for that now.